backend/src/main/java/com/bakdata/conquery/resources/admin/rest/AdminProcessor.java

```diff
@@ -74,7 +74,17 @@ public void addRoles(List<Role> roles) {
 	 * @param role the role to delete
 	 */
 	public void deleteRole(Role role) {
-		AuthorizationHelper.deleteRole(storage, role);
```

Comment:
Der AuthorizationHelper war diese riesen Helper Klasse die alles um Auth vereint hat oder? Warum willst du die hier auflösen?

Reply:
Da der Storage jetzt in den Entities integriert ist, braucht man die ganzen Methoden nicht mehr

Reply:
Ich merke gerade, dass da doch noch viele Methoden sind, die hatte ich in einem späteren commit rausgeworfen.
Aber jetzt wo ich es auf diesen StorageUpdater umgemünzt habe und nicht mehr den Storage habe, kann ich leider nicht mehr aus den Entities auf den Storage zugreifen, um z.b. die Group-Mappings eines Users zu Löschen. Das muss ich mit nochmal über legen.